src/ReadTrainer/PlayText.js

```diff
@@ -11,71 +11,132 @@ import {
   FontIcon
 } from "react-md";
 
+import { HotKeys } from "react-hotkeys";
+
 import { makeLogger } from "../analytics";
 
 import "./ReadTrainer.css";
 
+const keyMap = {
+  show: "space",
+  correct: "left",
+  incorrect: "right"
+};
+
 const event = makeLogger("ReadTrainer");
 
 const PlayHiddenCard = inject("store")(
-  observer(({ store, onShow }) => (
-    <Card className="bottom-card">
-      <CardTitle
-        title="Listen"
-        subtitle={store.morse.playing ? "Playing..." : "Waiting..."}
-      />
-      <CardText>
-        <p>Decode the text and press 'Show' when ready</p>
-      </CardText>
-      <CardActions centered>
-        <Button
-          raised
-          primary
-          onClick={onShow}
-          iconEl={<FontIcon>visibility</FontIcon>}
-        >
-          Show
-        </Button>
-      </CardActions>
-    </Card>
-  ))
+  observer(
+    class PlayHiddenCard extends Component {
+      componentDidMount() {
+        document.getElementById("PlayHiddenCard").focus();
```

Comment:
FYI: I think the react way of doing this is to use a ref (https://reactjs.org/docs/refs-and-the-dom.html). I've never done it, though, and I'm not entirely sure how to. Feel free to just leave it as is.

Reply:
I will play around some more and see if I can get it to work. I don't recall trying to use React to give a non-form element focus.

Reply:
I got it to work. I have switched out the implementation.